tools/Normalise_Consent_Data.php

```diff
@@ -0,0 +1,307 @@
+<?php
+/**
+ * This script is written for a one time use only to facilitate the normalization of the consent feature of candidate parameters. 
```

Comment:
Should this go into `tools/single_use/`? @ridz1208 ?

Reply:
yes, probably